Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -29,6 +33,15 @@ clean_unit = (units) ->
 
 module.exports.createQuantity = (value,unit) ->
   new Quantity({value: value, unit: unit})
+  
+module.exports.parseQuantity = (str) ->
+  components = /(\d+\.?\d*)(.+)/.exec str
```

Comment:
Again, based on the spec, I think this is not entirely correct.  Irregardless of format, one notable issue is that this would not properly work with negative quantities.